.github/workflows/build-and-push-image-development.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Prepare tags
         id: prepare-tags
         run: |
-          DOCKER_IMAGE=${{ secrets.DEVELOPMENT_ACR_URL }}/academies-academisation-api
```

Comment:
That name doesn't seem anywhere near as descriptive.
Why the change?

Reply:
It's how I initially launched the infrastructure unfortunately :( ...
Changing the name to `academies-academisation-api` now, will cause resources to be recreated and we'll have to reconfigure the apps that are connecting to the api
`aca` was initially chosen, as that was how the service was being refered to